src/csharp/Microsoft.Spark.E2ETest/UdfTests/UdfComplexTypesTests.cs

```diff
@@ -219,14 +219,14 @@ public void TestUdfWithReturnAsRowType()
                 }
             }
 
-            // Generic row is a part of top-level column.
+            // Row is a part of top-level column.
             {
                 var schema = new StructType(new[]
                 {
                     new StructField("col1", new IntegerType())
                 });
                 Func<Column, Column> udf = Udf<string>(
-                    str => new GenericRow(new object[] { 111 }), schema);
+                    str => new Row(new object[] { 111 }), schema);

```

Comment:
This example is not good since this `Row` constructor is internal.

Reply:
Changed to `GenericRow`.